sklearn/cross_validation.py

```diff
@@ -1438,6 +1438,12 @@ def cross_val_score(estimator, X, y=None, scoring=None, cv=None, n_jobs=1,
     -------
     scores : array of float, shape=(len(list(cv)),)
         Array of scores of the estimator for each run of the cross validation.
+
+    See Also
+    ---------
+    :func:`sklearn.metrics.make_scorer`:
+        Make a scorer from a performance metric or loss function.
```

Comment:
I'm looking at the comment style [here](https://github.com/scikit-learn/scikit-learn/blob/51a765a/sklearn/feature_selection/univariate_selection.py#L442). Can you try removing the space before the right-most colon?